src/ORM/Connect/PDOConnector.php

```diff
@@ -140,6 +140,9 @@ public static function is_emulate_prepare()
         return self::config()->get('emulate_prepare');
     }
 
+    /**
+     * @deprecated Use native database drivers in favour of PDO. ' . 'https://github.com/silverstripe/silverstripe-framework/issues/8598
```

Comment:
```suggestion
     * @deprecated 4.5 Use native database drivers instead
```
Update the notice's message as well.

Reply:
Upated